pkg/core/native/policy.go

```diff
@@ -131,6 +133,8 @@ func newPolicy() *Policy {
 	desc = newDescriptor("postPersist", smartcontract.VoidType)
 	md = newMethodAndPrice(getOnPersistWrapper(postPersistBase), 0, smartcontract.AllowModifyStates)
 	p.AddMethod(md, desc, false)
+
+	p.blockedAccounts = make([]util.Uint160, 0)
```

Comment:
I think this can be a part of `Initialize`. You have `isValid == false` after this, so `blockedAccounts` won't be used until it's set by either `Initialize` or `GetBlockedAccountsInternal`.

Reply:
You're right, fixed.